### PR TITLE
Site Management Page: Coming Soon Tile: make it work in RTL mode

### LINKED ISSUE #68951: Site Management Page: Coming Soon Tile - make it work for RTL
### Details

A small item remaining from https://github.com/Automattic/wp-calypso/pull/68886

### Checklist

- [ ] The layout should be updated to support RTL languages like Hebrew.

### Related

https://github.com/Automattic/wp-calypso/pull/68886

### PR DESCRIPTION
**Pull request by mpkelly**
Fixes https://github.com/Automattic/wp-calypso/issues/68951

#### Proposed Changes

* Make the tile support RTL mode

**Note**: I figured this would be possible with CSS but I wasn't able to get it working after trying a bunch of things. So I opted for flipping the `x` coordinate manually. If you know of a better solution feel free to push a commit. 

image

#### Testing Instructions

* Try it with a RTL language like Hebrew.
* Verify it still works ok in LTR mode

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Related to #

### CODE REVIEW

**Comment by wojtekn** [comment]
@mpkelly I've noticed the unusual logic there:

* site uses Espanol, Calypso uses Hebrew, and "Coming Soon" is displayed in Espanol and RTL
* site uses English, Calypso uses Hebrew, and "Coming Soon" is displayed in Hebrew and RTL

It's odd to me to display Espanol text in RTL mode and to use site's language in one case and Calypso's in another case.

If we treat the Coming Soon Tile as a screenshot, I would expect that:
* it follows the LTR/RTL of the site, so for the site in Hebrew, I would expect RTL, and for the site in Espanol, I would expect LTR and not the direction of the Calypso interface
* the "Coming Soon" text is always written in the site's language, so for the site in Hebrew, I would expect "בקרוב", and for the site in English, I would expect "Coming Soon".

Is this inconsistency intentional?

**Comment by mpkelly** [comment]
Thanks for your review, @wojtekn 

> * site uses English, Calypso uses Hebrew, and "Coming Soon" is displayed in Hebrew and RTL

I tried to follow the existing logic here. The text in the tile footer also displays RTL even though it's English:

image

> * site uses Espanol, Calypso uses Hebrew, and "Coming Soon" is displayed in Espanol and RTL

This is a bug; thanks for finding it. If the site language matches the current language [here](https://github.com/Automattic/wp-calypso/blob/1d06d92f999461683ee4fcacb179d5cf6c11b923/client/sites-dashboard/components/sites-site-coming-soon.tsx#L46), then we should return.  We will still want to show Hebrew if no translation exists for the site lang, though, as per this [comment](https://github.com/Automattic/wp-calypso/pull/68886#discussion_r992922489).

**Comment by wojtekn** [comment]
> I tried to follow the existing logic here. The text in the tile footer also displays RTL even though it's English:

I think this part is ok - if I use Hebrew, I would expect that everything on the page besides external images uses RTL - and I would consider the screenshot content as an external image.

**Comment by p-jackson** [comment]
> If you know of a better solution feel free to push a commit.

I don't know of a better way like an equivalent of `inset-inline-start` but for SVG. But I have added the `direction` property to the `<SVG>` element. Previously it was just inheriting the text direction from the document, but this means that this little subsection of the DOM can have a different text direction.

I've also made sure that we're using the text direction of the site, not the current user.

**Spanish interface, Hebrew site**
es-interface-he-site

**Hebrew interface, English site**
he-interface-en-site

I also fixed a bug in 3cfc2e9 where the site language wasn't being used if the site was English.

**Review by p-jackson — APPROVED** [review]
I think this is good to go (if I do say so myself). Although I think we finish this discussion before merging: p1665672877103569-slack-C0347E545HR

**Comment by mpkelly** [comment]
Thanks for those changes,@p-jackson. I have deployed this now.

**Comment by wojtekn** [comment]
Nice, all cases look good for me @p-jackson @mpkelly 👍

**Pull request merged**
